{"name": "Авторизация курьера — проверка ошибок", "status": "passed", "description": "Проверка авторизации негативными сценариями", "steps": [{"name": "Сценарий: Неправильный пароль", "status": "passed", "start": 1757417942549, "stop": 1757417943254}, {"name": "Проверить код ответа", "status": "passed", "start": 1757417943254, "stop": 1757417943254}, {"name": "Проверить сообщение об ошибке", "status": "passed", "start": 1757417943254, "stop": 1757417943254}], "parameters": [{"name": "description", "value": "'Неправильный пароль'"}, {"name": "creds", "value": "{'login': 'login_user_456', 'password': 'wrong_pass'}"}, {"name": "expected_status", "value": "404"}, {"name": "expected_message", "value": "'Учетная запись не найдена'"}], "start": 1757417942549, "stop": 1757417943254, "uuid": "96bf9b8e-e5e7-42ee-89dd-615f8c953378", "historyId": "d40d110f8a6c5b6a7fb7090dde9e5280", "testCaseId": "19c43b3cacae3758579e377968a7e303", "fullName": "tests.test_login_courier.TestCourierLogin#test_login_errors", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_login_courier"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "Ksu"}, {"name": "thread", "value": "16324-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_login_courier"}], "titlePath": ["tests", "test_login_courier.py", "TestCourierLogin"]}